simplex-solver/resources/simplex-lib-gulp-task.js

```diff
@@ -0,0 +1,19 @@
+// Javascript lib
+gulp.task('js-lib', function() {
+  var appConfig = _.pick(config, 'env');
+  return combineWatch([
+    //gulp.src(['src/index.js', 'src/equation.js']),
+    gulp.src(['simplex-solver.inc.js']),
+    browserify({
+      debug: config.env.development,
+      insertGlobals: true,
+      insertGlobalVars: {
```

Comment:
Honestly, I don't know. I just copied a task in the original Gulpfile to be able to build the module as a library.